src/azure-cli/azure/cli/command_modules/vm/commands.py

```diff
@@ -412,7 +412,7 @@ def load_command_table(self, _):
         g.command('simulate-eviction', 'simulate_eviction', command_type=compute_vmss_vm_sdk, min_api='2019-12-01')
         g.custom_command('start', 'start_vmss', supports_no_wait=True)
         g.custom_command('stop', 'stop_vmss', supports_no_wait=True, validator=process_vm_vmss_stop)
-        g.generic_update_command('update', getter_name='get_vmss', setter_name='update_vmss', supports_no_wait=True, command_type=compute_custom, validator=validate_vmss_update_namespace)
+        g.generic_update_command('update', getter_name='get_vmss_modified', setter_name='update_vmss', supports_no_wait=True, command_type=compute_custom, validator=validate_vmss_update_namespace)
```

Comment:
This is a very special case, it's better to add a standalone test for it.

Reply:
Considering it needs to update the permission of user in a subscription, I'm OK to skip the test. Let's make sure the test is done in local.

Reply:
I test it manually

Reply:
long term we need to switch to patch mode to completely resolve the issue. I'll file an issue to track it.